stable-diffusion/sdxl-turbo/config.yaml

```diff
@@ -0,0 +1,25 @@
+environment_variables: {}
+external_package_dirs: []
+model_metadata:
+  example_model_input: {"prompt": "A tree in a field under the night sky"}
+  pretty_name: SDXL Turbo
+  avatar_url: https://cdn.baseten.co/production/static/stability.png
+  cover_image_url: https://cdn.baseten.co/production/static/sd.png
+  tags:
+  - image-generation
+model_name: SDXL Turbo
+python_version: py310
+requirements:
+  - torch==2.0.1
+  - transformers==4.35.2
+  - diffusers==0.23.1
+  - scipy==1.11.3
```

Comment:
where's the scipy req coming from?

Reply:
Looks like scipy is not needed. Good catch!